src/lib/__tests__/server-utilities.test.js

```diff
@@ -28,3 +30,45 @@ describe('md5Hash', () => {
     expect(fileNameHash).not.toBe(md5Hash(fileName));
   });
 });
+
+describe('compressJPEG', () => {
+  const filePath = './tmp/test.jpg';
+
+  beforeAll(() => {
+    if (!fs.existsSync('./tmp')) fs.mkdirSync('./tmp');
+
+    // Copy compressible JPEG to `./tmp` directory to simulate JPEG upload
+    return copyFile('./src/lib/__tests__/assets/compression-test.jpg', filePath);
```

Comment:
Exactly what I wanted, this is really good, and without a doubt more elegant than using `done()` :)